snuba/web/views.py

```diff
@@ -404,7 +407,6 @@ def eventstream(*, dataset: Dataset):
         )
 
         type_ = record[1]
-        metrics = DummyMetricsBackend()
```

Comment:
Unrelated-ish, but shadowing here is not really necessary.

Reply:
Does this actually change anything for tests?

Reply:
It depends on what settings file is used — by default, it will use the default metrics backend, in which case it'll cause some log spam but won't throw. If the no `DATADOG_{HOST,PORT}` are provided, it'll have the same behavior it did before: https://github.com/getsentry/snuba/blob/00598cc5cd3d4bfc5c05f1cfbc1009c88231cebf/snuba/util.py#L294-L323

In the worst case it's all bark and no bite